File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L5/08/Kasus3L5.ts
import { Scanner } from 'readline-sync';

function hitungBMI(weight: number, height: number): number {
  return weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254));
}

function BMIKategori(BMI: number): void {
  console.log(`BMI is ${BMI}`);
  if (BMI < 18.5) {
    console.log('Underweight');
  } else if (BMI < 25) {
    console.log('Normal');
  } else if (BMI < 30) {
    console.log('Overweight');
  } else {
    console.log('Obese');
  }
}

function main(): void {
  const inp = new Scanner();
  console.log('Enter weight in pounds: ');
  const berat = inp.nextDouble();

  console.log('Enter feet: ');
  const kaki = inp.nextDouble();
  console.log('Enter inches: ');
  const inch = inp.nextDouble();

  BMIKategori(hitungBMI(berat, kaki * 12 + inch));
}

main();

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-03/plagiarized/L5/08/Kasus3L5.kt
package IR

import java.util.Scanner

fun main() {
    println("Enter weight in pounds: ")
    val inp = Scanner(System.`in`)
    val berat = inp.nextDouble()

    println("Enter feet: ")
    val kaki = inp.nextDouble()
    println("Enter inches: ")
    val inch = inp.nextDouble()

    BMIKategori(hitungBMI(berat, kaki * 12 + inch))
}

private fun hitungBMI(weight: Double, height: Double): Double {
    return weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254))
}

private fun BMIKategori(BMI: Double) {
    println("BMI is $BMI")
    when {
        BMI < 18.5 -> println("Underweight")
        BMI < 25 -> println("Normal")
        BMI < 30 -> println("Overweight")
        else -> println("Obese")
    }
}

